apps/jupyter/extensions/notebook-runner/src/iframe-api.ts

```diff
@@ -265,6 +282,15 @@ export class EmbeddedPythonCallbacks {
         `${logModule} RPC: ${request.method} failed with error:`,
         e,
       );
```

Comment:
^ above

What if the format is unknown?

Reply:
This is still an open question: You almost always want a `default` clause for a switch statement. In case you never expect any other value, you can simply throw in there but then at least nothing weird happens.

Reply:
Yes I saw! Still, it is usually good practice to have a fallback `default` branch.
Imagine you add a new enum value but forget to change this code here - then the behavior is suddenly weird. If we throw, you will immediately see that something must be changed. Does that make sense?